code/index.html

```diff
@@ -28,9 +28,12 @@
       </article>
     </section>
 
```

Comment:
Is this empty line for readability? Or can it be deleted to keep consistency with the above code where there are not empty lines?